setup.cfg

```diff
@@ -37,8 +37,5 @@ url = http://sunpy.org/
 edit_on_github = True
 github_project = sunpy/sunpy
 
-[pycodestyle]
-max_line_length = 100
-
-[yapf]
```

Comment:
whyyyy? This is very useful.

Reply:
Meh to YAPF. I don't think we plan to use it in any meaningful way.

Reply:
I still use it locally sometimes though.